src/RequestProcessor.cs

```diff
@@ -70,6 +99,27 @@ internal StreamingMessage ProcessWorkerInitRequest(StreamingMessage request)
             return response;
         }
 
+        internal StreamingMessage ProcessWorkerTerminateRequest(StreamingMessage request)
+        {
+            return null;
+        }
+
+        internal StreamingMessage ProcessWorkerStatusRequest(StreamingMessage request)
+        {
+            // WorkerStatusResponse type says that it is not used but this will create an empty one anyway to return to the host
```

Comment:
@pragnagopa can you comment on this message type?

Reply:
This is a placeholder message. Empty is ok for now. When we implement heartbeats, we use this message.